### ISSUE TITLE
Hot Controls doesn't work with turbo-permanent

### ISSUE THREAD

**Issue by Metallizzer** [octobercms/ajax#26]
Hello, octobercms@1.3.0 from npm

Hot Controls doesn't work if we attach it to element inside element with data-turbo-permanent enabled.
It throws an error "Error connecting controller" after PJAX page load.
It works only first time during regular page load

Code example:
```html
<header class="header" id="header" data-turbo-permanent>
    ...
    <form class="search-form" action="{{ 'search'|page }}" data-control="search">
        ...
    </form>
    ...
</header>

<script>
    oc.registerControl(
        "search",
        class extends oc.ControlBase {
            init() {
                this.$term = this.element.querySelector("input.form-control");
            }

            connect() {
                this.listen("submit", function (e) {
                    e.preventDefault();

                    var term = this.$term.value.trim();

                    if (!term.length || this.$term.defaultValue == term) {
                        return;
                    }

                    oc.visit(this.element.action + "?q=" + term);
                });
            }
        }
    );
</script>
```
![image](https://github.com/octobercms/ajax/assets/691719/ecaec2db-29c9-4c76-81a2-06f1873b0c1c)

**Comment by daftspunk** [comment]
Hi @Metallizzer 

Coincidentally we fixed this yesterday (029be503196d1b2b6bc24204967b0903eea2d037).

v1.3.1 has been published just now with the fix. Please try it and let me know.

**Comment by Metallizzer** [comment]
Yes, now it works. Thanks @daftspunk
